images/manageiq-webserver-worker/Dockerfile

```diff
@@ -9,23 +9,24 @@ ARG RPM_PREFIX=manageiq
 LABEL name="manageiq-webserver-worker" \
       summary="ManageIQ web server worker image"
 
-RUN dnf -y --setopt=tsflags=nodocs install ${RPM_PREFIX}-ui && \
-    dnf clean all && \
-    rm -rf /var/cache/dnf
-
 COPY container-assets/service-worker-entrypoint /usr/local/bin
 
-RUN rm -f /etc/httpd/conf.d/* && \
-    sed -i '/^Listen 80/d' /etc/httpd/conf/httpd.conf
-
-# Configure httpd to run without root privileges
-RUN chgrp root /var/run/httpd && chmod g+rwx /var/run/httpd && \
-    chgrp root /var/log/httpd && chmod g+rwx /var/log/httpd
-
+RUN dnf -y --setopt=tsflags=nodocs install \
+      ${RPM_PREFIX}-ui && \
+    clean_dnf_rpm && \
+    # Remove httpd default settings
+    rm -f /etc/httpd/conf.d/* && \
+    sed -i '/^Listen 80/d' /etc/httpd/conf/httpd.conf && \
+    # Configure httpd to run without root privileges
+    chgrp root /var/run/httpd && \
+    chmod g+rwx /var/run/httpd && \
+    chgrp root /var/log/httpd && \
+    chmod g+rwx /var/log/httpd
+
+# Build the RPM manifest
```

Comment:
Any reason to merge the other `RUN` steps but not this one?

Reply:
It doesn't save any space to split them up, though admittedly it would drop the need to duplicate the clean_dnf_rpm call.  I'd like to leave it for now.